### ISSUE TITLE
Telegram - Various commands not working correctly

### ISSUE THREAD

**Issue by zappra** [CoinAlpha/hummingbot#3062]
**Describe the bug**
If you enter the command "trades" via the bot UI it will report trades for the last 24 hours, ie using default values for the positional arguments

If you enter the same command via telegram the following error is generated:

2021-03-10 07:27:39,622 - 1 - hummingbot.client.ui.hummingbot_cli - ERROR - Unhandled error in prompt_toolkit: trades() missing 3 required positional arguments: 'days', 'market', and 'open_order_markets'

**Release version**
dev-0.37.0

**Comment by zappra** [comment]
Some or all of the arguments to the command function are missing when called via call_soon_threadsafe

Issue also affects history, pnl, ticker, order_book, stop and start commands

**Comment by john-hummingbot-support** [comment]
This is to confirm the issue posted above, tho other commands are working on my end especially those commands with buttons. attach is the whole log file.
[logs_algo.log](https://github.com/CoinAlpha/hummingbot/files/6125225/logs_algo.log)


this is for order_book command

`2021-03-11 18:09:12,643 - 1 - hummingbot.client.hummingbot_application - ERROR - There is no current event loop in thread 'ThreadPoolExecutor-0_2'.
Traceback (most recent call last):
  File "/home/hummingbot/hummingbot/client/hummingbot_application.py", line 141, in _handle_command
    f(**kwargs)
  File "/home/hummingbot/hummingbot/client/command/order_book_command.py", line 14, in order_book
    safe_ensure_future(self.show_order_book(lines, exchange, market, live))
  File "/home/hummingbot/hummingbot/core/utils/async_utils.py", line 17, in safe_ensure_future
    return asyncio.ensure_future(safe_wrapper(coro), *args, **kwargs)
  File "/home/hummingbot/miniconda3/envs/hummingbot/lib/python3.8/asyncio/tasks.py", line 660, in ensure_future
    loop = events.get_event_loop()
  File "/home/hummingbot/miniconda3/envs/hummingbot/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'ThreadPoolExecutor-0_2'.`

for pnl command

`2021-03-11 18:09:34,139 - 1 - hummingbot.client.ui.hummingbot_cli - ERROR - Unhandled error in prompt_toolkit: trades() missing 3 required positional arguments: 'days', 'market', and 'open_order_markets'
Traceback (most recent call last):
  File "/home/hummingbot/miniconda3/envs/hummingbot/lib/python3.8/asyncio/events.py", line 81, in _run
    self._context.run(self._callback, *self._args)
TypeError: trades() missing 3 required positional arguments: 'days', 'market', and 'open_order_markets'`

for ticker command

`2021-03-11 18:09:39,153 - 1 - hummingbot.client.hummingbot_application - ERROR - There is no current event loop in thread 'ThreadPoolExecutor-0_0'.
Traceback (most recent call last):
  File "/home/hummingbot/hummingbot/client/hummingbot_application.py", line 141, in _handle_command
    f(**kwargs)
  File "/home/hummingbot/hummingbot/client/command/ticker_command.py", line 15, in ticker
    safe_ensure_future(self.show_ticker(live, exchange, market))
  File "/home/hummingbot/hummingbot/core/utils/async_utils.py", line 17, in safe_ensure_future
    return asyncio.ensure_future(safe_wrapper(coro), *args, **kwargs)
  File "/home/hummingbot/miniconda3/envs/hummingbot/lib/python3.8/asyncio/tasks.py", line 660, in ensure_future
    loop = events.get_event_loop()
  File "/home/hummingbot/miniconda3/envs/hummingbot/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'ThreadPoolExecutor-0_0'.`

for trades command

`2021-03-11 18:25:48,584 - 1 - hummingbot.client.ui.hummingbot_cli - ERROR - Unhandled error in prompt_toolkit: trades() missing 3 required positional arguments: 'days', 'market', and 'open_order_markets'
Traceback (most recent call last):
  File "/home/hummingbot/miniconda3/envs/hummingbot/lib/python3.8/asyncio/events.py", line 81, in _run
    self._context.run(self._callback, *self._args)
TypeError: trades() missing 3 required positional arguments: 'days', 'market', and 'open_order_markets'
`

**Comment by sidmontanez** [comment]
Same goes on my end. Only the available commands that is on Telegram are the one that's working properly. 

![image](https://user-images.githubusercontent.com/78310937/110840802-1e71ed00-82e0-11eb-9c00-3956cf4dfef7.png)


Here's some result when I run the `ticker`,`order_book` & `pnl` through telegram. 
![image](https://user-images.githubusercontent.com/78310937/110840720-0732ff80-82e0-11eb-9190-38bc0473fda3.png)

And here's some sample logs when the error occured

``` 
2021-03-11 19:04:11,837 - 1 - hummingbot.client.hummingbot_application - ERROR - There is no current event loop in thread 'ThreadPoolExecutor-0_1'.
Traceback (most recent call last):
  File "/home/hummingbot/hummingbot/client/hummingbot_application.py", line 141, in _handle_command
    f(**kwargs)
  File "/home/hummingbot/hummingbot/client/command/order_book_command.py", line 14, in order_book
    safe_ensure_future(self.show_order_book(lines, exchange, market, live))
  File "/home/hummingbot/hummingbot/core/utils/async_utils.py", line 17, in safe_ensure_future
    return asyncio.ensure_future(safe_wrapper(coro), *args, **kwargs)
  File "/home/hummingbot/miniconda3/envs/hummingbot/lib/python3.8/asyncio/tasks.py", line 660, in ensure_future
    loop = events.get_event_loop()
  File "/home/hummingbot/miniconda3/envs/hummingbot/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'ThreadPoolExecutor-0_1'.

**Comment by zappra** [comment]
start command works, but doesn't pass restore parameter correctly (will always be the default, false)
stop command works, but doesn't pass skip_order_cancellation correctly (will always be the default, false)
history command works, again with only default values for days, verbose and precision

Depending on Telegram platform, you might need to turn off 'Replace Emoji Automatically' in settings in order to send '--' correctly.

**Comment by zappra** [comment]
For example, on iOS, attempting to send 'start --restore' won't work, it replaces the '--' with a long dash which generates an "unrecognised arguments' response. I can't find the emoji setting on iOS.

**Comment by dennisocana** [comment]
Fixed in version 0.44 release. Thanks @zappra for contributing 👍

**Comment by dennisocana** [comment]
We are including this fix in tomorrow's version 0.43 release instead.
